Debug App/Sources/View Controllers/New UI/MerchantHeadlessCheckoutNolPayViewController.swift

```diff
@@ -13,7 +13,7 @@ import IQKeyboardManagerSwift
 #if canImport(PrimerNolPaySDK)
 class MerchantHeadlessCheckoutNolPayViewController: UIViewController {
     
-    private var nolPayManager: PrimerHeadlessUniversalCheckout.PrimerHeadlessNolPayManager!
+    private var nolPayManager: PrimerSDK.PrimerHeadlessUniversalCheckout.PrimerHeadlessNolPayManager!
```

Comment:
Whats the driver behind this change?

Reply:
I know its the Debug app and less important here - but wondering if this Syntax will be required for the merchant?

Reply:
Xcode autocomplete... I removed it.